app/routes/form.$version.docs.$.tsx

```diff
@@ -17,7 +17,7 @@ export const loader = async (context: LoaderFunctionArgs) => {
   })
 }
 
-export const meta: MetaFunction = ({ data }) => {
+export const meta: MetaFunction<typeof loader> = ({ data }) => {
```

Comment:
I fixed the type errors around the `meta` functions in the files I worked on. Most of the remaining errors of this kind are getting fixed in https://github.com/TanStack/tanstack.com/pull/135.